src/NServiceBus.AcceptanceTests/Node/When_sending_to_a_node.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace NServiceBus.AcceptanceTests.Node
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Pipeline;
+    using NServiceBus.Pipeline.Contexts;
+    using NServiceBus.Routing;
+    using NServiceBus.Transports;
+    using NUnit.Framework;
+
+    public class When_sending_to_a_node: NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_receive_the_message()
+        {
+            var nodeConfig = new MicroEndpointConfiguration(new LogicalAddress(new EndpointInstance("SendingToANode.Receiver", null, null)));
```

Comment:
Don't like passing an endpoint instance. Some kind of "receive address" would make more sense